src/connector.rs

```diff
@@ -0,0 +1,90 @@
+//! Utility methods for instantiating common connectors for clients.
+extern crate hyper_tls;
+extern crate tokio_core;
+extern crate openssl;
+extern crate native_tls;
+
+use std::path::Path;
+
+use hyper;
+use self::tokio_core::reactor::Handle;
+
+/// Returns a function which creates an http-connector. Used for instantiating
+/// clients with custom connectors
+pub fn http_connector() -> Box<Fn(&Handle) -> hyper::client::HttpConnector + Send + Sync> {
+    Box::new(move |handle: &Handle| {
+        hyper::client::HttpConnector::new(4, handle)
+    })
+}
+/// Returns a function which creates an https-connector
```

Comment:
Minor, but don't we normally leave blank lines between functions?

Reply:
Yup, we do.